src/views/GraphsGrid.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="d-flex flex-column h-100">
+    <div class="graphs-grid">
+      <div v-for="pair in botStore.activeBot.whitelist" class="grid">
```

Comment:
I don't think that's a good idea (especially not in webserver mode - where there is no whitelist (the whitelist request will fail)- so this is basically pointless / nonfunctional in webserver mode).

instead, it should be a list (most likely based on the whitelist) - where users can select "i want to see this, this and that pair" (maybe a select all button - but ALL pairs shouldn't be the default).
Now what list we should use in webserver mode is a different topic ... one i'll need to look into further.

I can have a pairlist of 70 pairs - but showing me 70 charts will not allow me to compare anything, unless the 2 "comparing" pairs happen to be next to each other - which is pretty unlikely.

Reply:
The same time the usual chart is that friendly in terms of UX. 

In my working scenariou I wan't to see how strategy behaves across all 100 pairs limited on a single chart on a screen, while an asset can only be changed from the dropdown - meaning I need to click 100 times.

Reply:
well i do understand this - that's the "select all" scenario (which i'd still limit to some sane number of pairs though to avoid memory problems).

by default, I'd however like to only show a selected few. 
That'll also reduce the amount of data necessary - and keep the UI responsive.

Reply:
> by default, I'd however like to only show a selected few.

Agree, the whole collection of pairs is to be displayed in a paginated manner and loaded dynamically. say 10 items per page.